Failure in wallet_import_rescan.py

Appveyor build failed for #15849 in https://ci.appveyor.com/project/DrahtBot/bitcoin/builds/23959262:

```
 test  2019-04-19T01:01:12.401000Z TestFramework (ERROR): Assertion failed 
                                   Traceback (most recent call last):
                                     File "C:\projects\bitcoin\test\functional\test_framework\test_framework.py", line 175, in main
                                       self.run_test()
                                     File "C:\projects\bitcoin/test/functional/wallet_import_rescan.py", line 185, in run_test
                                       variant.check()
                                     File "C:\projects\bitcoin/test/functional/wallet_import_rescan.py", line 78, in check
                                       assert_equal(len(txs), self.expected_txs)
                                     File "C:\projects\bitcoin\test\functional\test_framework\util.py", line 39, in assert_equal
                                       raise AssertionError("not(%s)" % " == ".join(str(arg) for arg in (thing1, thing2) + args))
                                   AssertionError: not(1 == 0)
 test  2019-04-19T01:01:12.405000Z TestFramework (DEBUG): Closing down network thread 
 test  2019-04-19T01:01:12.455000Z TestFramework (INFO): Stopping nodes 
 test  2019-04-19T01:01:12.455000Z TestFramework.node0 (DEBUG): Stopping node 
```

--- a/test/functional/wallet_import_rescan.py
+++ b/test/functional/wallet_import_rescan.py
@@ -166,11 +166,12 @@ class ImportRescanTest(BitcoinTestFramework):
         timestamp = self.nodes[0].getblockheader(self.nodes[0].getbestblockhash())["time"]
         set_node_times(self.nodes, timestamp + TIMESTAMP_WINDOW + 1)
         self.nodes[0].generate(1)
-        self.sync_blocks()
+        self.sync_all()
 
         # For each variation of wallet key import, invoke the import RPC and
         # check the results from getbalance and listtransactions.
         for variant in IMPORT_VARIANTS:
+            self.log.info('Run import for variant {}'.format(variant))
             variant.expect_disabled = variant.rescan == Rescan.yes and variant.prune and variant.call == Call.single
             expect_rescan = variant.rescan == Rescan.yes and not variant.expect_disabled
             variant.node = self.nodes[2 + IMPORT_NODES.index(ImportNode(variant.prune, expect_rescan))]
@@ -192,10 +193,11 @@ class ImportRescanTest(BitcoinTestFramework):
         # Generate a block containing the new transactions.
         self.nodes[0].generate(1)
         assert_equal(self.nodes[0].getrawmempool(), [])
-        self.sync_blocks()
+        self.sync_all()
 
         # Check the latest results from getbalance and listtransactions.
         for variant in IMPORT_VARIANTS:
+            self.log.info('Run check for variant {}'.format(variant))
             if not variant.expect_disabled:
                 variant.expected_balance += variant.sent_amount
                 variant.expected_txs += 1

